test/boost/group0_test.cc

```diff
@@ -60,14 +60,14 @@ SEASTAR_TEST_CASE(test_abort_server_on_background_error) {
         auto check_error = [](const raft::stopped_error& e) {
             return e.what() == sstring("Raft instance is stopped, reason: \"background error, std::runtime_error (store_log_entries/test-failure)\"");
         };
-        BOOST_REQUIRE_EQUAL(get_status(), 1);
+        BOOST_REQUIRE_EQUAL(get_status(), 0);
         BOOST_CHECK_EXCEPTION(co_await perform_schema_change(), raft::stopped_error, check_error);
-        BOOST_REQUIRE_EQUAL(get_status(), 2);
+        BOOST_REQUIRE_EQUAL(get_status(), 1);
         BOOST_CHECK_EXCEPTION(co_await perform_schema_change(), raft::stopped_error, check_error);
-        BOOST_REQUIRE_EQUAL(get_status(), 2);
+        BOOST_REQUIRE_EQUAL(get_status(), 1);
```

Comment:
This test is in fact evidence why the status numbers must not be modified.

Reply:
aye